suites/smokeTests/brainTests.js

```diff
@@ -0,0 +1,200 @@
+Feature("Brain Promotion Smoke Test Suite");
+
+const chai = require('chai');
+const {interactive, ifInteractive} = require('../../utils/interactive.js');
+const expect = chai.expect;
+
+//@toBeAutomated
+Scenario('Login with Synapse first time @manual', ifInteractive(
```

Comment:
@atharvar28 , just a quick heads-up:
Since you are picking the specific set of executable tests that comprise Brain's smoke tests, I think we should leverage the 'at' (@) notation (CodeceptJS tag) to facilitate the execution of the test suite.

I have to do the same for DataSTAGE and DCF, a per:
> PXP-3839: datastage smoke test suite
> PXP-4465: dcf smoke test suite

Check the ticket's description.

Reply:
So maybe we could use `@brain` here? So it would contain both `@manual @brain` ?
What do you think?

Reply:
I am more inclined towards using `@brainsmoke` to make a clear indication of what is happening in a particular test run @themarcelor - just a thought! The annotations to test scenarios can keep going up and it would be difficult to come back once the process of annotations is on full throttle. but we can discuss later!

Reply:
awesome. And for common / overlapping ones we can annotate as @core 
@haraprasadj , what do you reckon?

Reply:
we can just have @smoke, @brain, @dcp etc. Some tests will have multiple annotations (@brain and @smoke) then we can test all kinds of combinations - all brain tests, smoke tests for brain etc.

Reply:
That also makes sense...
so it would be like:
```
`@brain @smoke`
```
And then we just grep whatever we want:
e.g.,
```
% RUNNING_LOCAL=true NAMESPACE="jenkins-genomel" npm test -- --verbose --grep '@smoke|@dcf'

> gen3-qa@0.1.0 test /Users/marcelocostarodrigues/workspace/gen3-qa
> codeceptjs run --plugins allure "--verbose" "--grep" "@smoke|@dcf"

jenkins-genomel
true
Running against k8s cluster.
Start setup
NAMESPACE: jenkins-genomel
HOSTNAME: jenkins-genomel.planx-pla.net
Tags:
[ '@smoke', '@dcf' ]
```

see?

[ '@smoke', '@dcf' ]
--